components/ui/project-form.tsx

```diff
@@ -43,50 +45,48 @@ export default function ProjectForm({ defaultValues }: Props) {
       const modifiedData = {
         ...data,
         id: defaultValues?.id,
-      }
+      };
       editProject(modifiedData)
-      .catch(err => console.error('project edit error'))
-      .then(() => console.log('successfully edited'))
-      . catch(() => console.log('promise catched'));
-    } else{
+        .catch(() => console.error('project edit error'))
+        .then(() => console.log('successfully edited'))
+        .catch(() => console.log('promise catched'));
+    } else {
       addProject(data)
-      .catch(err => console.error('project add error'))
-      .then(() => console.log('successfully added'))
-      .catch(() => console.log('promise catched'));
+        .catch(() => console.error('project add error'))
+        .then(() => console.log('successfully added'))
+        .catch(() => console.log('promise catched'));
     }
     router.push('/projects');
```

Comment:
Instead of client-side router-push, why not just `redirect` within the action code?

Reply:
fix: cddb6ff